### DISCUSSION TITLE
Cannot enable Actions on any repo

### DISCUSSION THREAD

**Discussion by towerundersiege** [community/community#66640] [Actions] [answered]
### Select Topic Area

Question

### Body

On existing repos, or new ones, I cannot enable actions. To confirm this I created a new repo, used the suggestion in the actions page to add a new a blank.yaml file in workflows to see if starting fresh would work, but no. Instead I get the option to "Enable Actions of this repository" but clicking that just returns the message "Unable to enable Actions for this repository."

Any ideas? From what I've read, actions should be available by default.

![image](https://github.com/community/community/assets/73253649/d8374875-aa91-4629-95b9-19cc2717af50)

**Comment by towerundersiege** [comment, marked as answer]
In the event anyone encounters this issue. My account had been flagged for using an aliased email address (in my case SimpleLogin) and so I could not use the Actions functionality. I had to remove the email in question and raise a ticket to be unflagged at https://support.github.com/request

**Reply by ghost** [reply]
I guessed so, perhaps i will need to sign up again.

**Comment by afshanbhat** [comment]
I am getting Unable to enable Actions for this repository...how can i resolve it
